registry/storage/vacuum.go

```diff
@@ -81,7 +82,26 @@ func (v Vacuum) RemoveManifest(name string, dgst digest.Digest, tags []string) e
 		return err
 	}
 	dcontext.GetLogger(v.ctx).Infof("deleting manifest: %s", manifestPath)
-	return v.driver.Delete(v.ctx, manifestPath)
+	err = v.driver.Delete(v.ctx, manifestPath)
+	if err != nil {
+		return err
+	}
+
+	referrerRootPath, err := pathFor(referrersRootPathSpec{name: name})
```

Comment:
For example: if Vacuum is configured with options for extensions, then it could optionally invoke the extension here to perform any clean up it needs to